starport_template/lib/widgets/sign_transaction_item.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:cosmos_ui_components/cosmos_text_theme.dart';
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+import 'package:starport_template/entities/balance.dart';
+
+class SignTransactionItem extends StatelessWidget {
+  const SignTransactionItem({
+    Key? key,
+    required this.text,
+    required this.balance,
+    required this.amount,
+  }) : super(key: key);
+
+  final String text;
+  final double amount;
+  final Balance balance;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = CosmosTheme.of(context);
+    return Padding(
+      padding: EdgeInsets.symmetric(horizontal: theme.spacingL),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Text(text, style: CosmosTextTheme.titleS),
+          Row(
```

Comment:
Row inside a Row is problematic from the layout performance perspective and might lead to crashes of "layout was given an infnite width`. What you're trying to accomplish here can be done with:
```dart
Row(
    children: [
        Text(text),
        Spacer(),
        Text(amount),
        SizedBox(),
        CosmosTokenAvatar(),
    ]
)
```

Reply:
Updated